libs/components/src/lib/empty-state/empty-state.scss

```
@@ -1,12 +1,28 @@
 @use '../../../../../dist/libs/tokens/scss/tokens.constants' as constants;
 @use '../../../../shared/src/lib/sass/mixins/border-radius' as
 	border-radius-variables;
+@use '../../../../shared/src/lib/sass/mixins/connotation/config' with (
+	$connotations: accent cta success alert warning information announcement,
+	$shades: firm firm-all pale contrast soft,
+	$default: accent
+);
+@use '../../../../shared/src/lib/sass/mixins/connotation' as connotation;
+@use '../../../../shared/src/lib/sass/mixins/appearance/config' as
+	appearance-config with (
+	$appearances: duotone,
+	$states: idle,
+	$default: duotone
+);
+@use '../../../../shared/src/lib/sass/mixins/appearance' as appearance;
 
 :host {
 	display: block;
 }
 
 .base {
+	@include connotation.connotation(badge);
+	@include appearance.appearance;
```

IMO, it is not right to use appearance mixin here. Design didn't use appearance so why should the code? The mixed colors end up different from the design 

```suggestion
	@include connotation.connotation(empty-state);
```

---

the badge is a huge mistake of mine and I'll fix it.

But we are always using connotation and appearance together - see alert & progress.
On the other hand we have icons that do not use it.
I wonder what is the right way to do that.
Do we want all components that has one appearance, no states to use connotation directly or use it via appearances...
If I'm not using connotation directly - I can use whatever shades I want, is it good for the design-system or should we align to appearances?
@AyalaBu @TaylorJ76 
WDYT?

---

I removed the appearance and created a ticket to align other component to the same convention: https://jira.vonage.com/browse/VIV-1960